util/issue_commands.go

```diff
@@ -1,59 +1,39 @@
 package util
 
 import (
-	"bytes"
-	"errors"
-	"os/exec"
-	"strconv"
-
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/planetmint/planetmint-go/config"
+	"github.com/planetmint/planetmint-go/lib"
+	daotypes "github.com/planetmint/planetmint-go/x/dao/types"
 )
 
-func InitRDDLReissuanceProcess(ctx sdk.Context, proposerAddress string, txUnsigned string, blockHeight int64) error {
+func buildSignBroadcastTx(ctx sdk.Context, sendingValidatorAddress string, msg sdk.Msg) (err error) {
+	logger := ctx.Logger()
+	addr := sdk.MustAccAddressFromBech32(sendingValidatorAddress)
+	txBytes, txJSON, err := lib.BuildAndSignTx(addr, msg)
+	if err != nil {
+		return
+	}
+	logger.Debug("REISSUE: tx: " + txJSON)
+	_, err = lib.BroadcastTx(txBytes)
```

Comment:
Is this call result in an async call or at least issued in a non-blocking way?
I ask that as if not, we will have difficulties debugging and analysing the workflows within the planetmint-go service.

Reply:
BroadcastMode is the default behavior: `BROADCAST_MODE_SYNC`

Reply:
and then it is a `http.Post`

Reply:
"each HTTP request is handled in it's own go routine" https://stackoverflow.com/a/52157993